Makefile

```diff
@@ -236,52 +236,35 @@ clean:
 	hack/make-rules/clean.sh
 endif
 
+VERSION ?= $(shell git describe --tags)
 
-QEMU_ARCH ?= x86_64
-ARCH ?= amd64
-IMAGE_TAG ?= $(shell git describe --tags)
-GO_LDFLAGS='$(shell hack/make-rules/version.sh)'
+IMAGES=cloudcore \
+  admission \
+  edgecore \
+  edgesite \
+  csidriver \
+  bluetooth
 
-.PHONY: cloudimage
-cloudimage:
-	docker build --build-arg GO_LDFLAGS=${GO_LDFLAGS} -t kubeedge/cloudcore:${IMAGE_TAG} -f build/cloud/Dockerfile .
-
-.PHONY: admissionimage
-admissionimage:
-	docker build --build-arg GO_LDFLAGS=${GO_LDFLAGS} -t kubeedge/admission:${IMAGE_TAG} -f build/admission/Dockerfile .
-
-.PHONY: csidriverimage
-csidriverimage:
-	docker build --build-arg GO_LDFLAGS=${GO_LDFLAGS} -t kubeedge/csidriver:${IMAGE_TAG} -f build/csidriver/Dockerfile .
-
-.PHONY: edgeimage
-edgeimage:
-	mkdir -p ./build/edge/tmp
-	rm -rf ./build/edge/tmp/*
-	curl -L -o ./build/edge/tmp/qemu-${QEMU_ARCH}-static.tar.gz https://github.com/multiarch/qemu-user-static/releases/download/v3.0.0/qemu-${QEMU_ARCH}-static.tar.gz
-	tar -xzf ./build/edge/tmp/qemu-${QEMU_ARCH}-static.tar.gz -C ./build/edge/tmp
-	docker build -t kubeedge/edgecore:${IMAGE_TAG} \
-	--build-arg GO_LDFLAGS=${GO_LDFLAGS} \
-	--build-arg BUILD_FROM=${ARCH}/golang:1.13.8-alpine3.10 \
-	--build-arg RUN_FROM=${ARCH}/docker:dind \
-	-f build/edge/Dockerfile .
-
-.PHONY: edgesiteimage
-edgesiteimage:
-	mkdir -p ./build/edgesite/tmp
-	rm -rf ./build/edgesite/tmp/*
-	curl -L -o ./build/edgesite/tmp/qemu-${QEMU_ARCH}-static.tar.gz https://github.com/multiarch/qemu-user-static/releases/download/v3.0.0/qemu-${QEMU_ARCH}-static.tar.gz
-	tar -xzf ./build/edgesite/tmp/qemu-${QEMU_ARCH}-static.tar.gz -C ./build/edgesite/tmp
-	docker build -t kubeedge/edgesite:${IMAGE_TAG} \
-	--build-arg GO_LDFLAGS=${GO_LDFLAGS} \
-	--build-arg BUILD_FROM=${ARCH}/golang:1.13.8-alpine3.10 \
-	--build-arg RUN_FROM=${ARCH}/docker:dind \
-	-f build/edgesite/Dockerfile .
```

Comment:
How can I achieve that If I just want to build a specific image?

Reply:
Like `make image WHAT=cloudcore`.

```
# Build images.
#
# Args:
#   WHAT: component name to build. support: cloudcore admission edgecore edgesite csidriver bluetooth
#         If not specified, "everything" will be built.
#
# Example:
#   make image
#   make image HELP=y
#   make image WHAT=cloudcore
```